build.sbt

```diff
@@ -20,51 +15,56 @@ resolvers ++= Seq(
 
 val awsVersion = "1.11.759"
 
-libraryDependencies ++= Seq(
-    "com.google.code.findbugs" % "jsr305" % "2.0.0",
-    "com.amazonaws" % "aws-java-sdk-dynamodb" % awsVersion,
-    "com.amazonaws" % "aws-java-sdk-ec2" % awsVersion,
-    "com.amazonaws" % "aws-java-sdk-iam" % awsVersion,
-    "com.amazonaws" % "aws-java-sdk-sts" % awsVersion,
-    "com.amazonaws" % "aws-java-sdk-autoscaling" % awsVersion,
-    "com.amazonaws" % "aws-java-sdk-s3" % awsVersion,
-    "com.amazonaws" % "aws-java-sdk-lambda" % awsVersion,
-    "com.amazonaws" % "aws-java-sdk-acm" % awsVersion,
-    "com.amazonaws" % "aws-java-sdk-route53" % awsVersion,
-    "com.amazonaws" % "aws-java-sdk-elasticloadbalancing" % awsVersion,
-
-    "com.gu" %% "management-play" % "8.0",
-    "com.typesafe.akka" %% "akka-agent" % "2.4.1",
-    "com.typesafe.akka" %% "akka-slf4j" % "2.4.1",
-    filters,
-    specs2 % "test"
-)
-
-scalacOptions ++= Seq("-feature")
-
-javaOptions in Universal ++= Seq(
-  s"-Dpidfile.path=/dev/null",
-  "-J-XX:MaxRAMFraction=2",
-  "-J-XX:InitialRAMFraction=2",
-  "-J-XX:MaxMetaspaceSize=300m",
-  "-J-XX:+PrintGCDetails",
-  "-J-XX:+PrintGCDateStamps",
-  s"-J-Xloggc:/var/log/${packageName.value}/gc.log"
-)
-
 lazy val root = (project in file("."))
-  .enablePlugins(PlayScala, RiffRaffArtifact, UniversalPlugin)
+  .enablePlugins(PlayScala, RiffRaffArtifact, JDebPackaging, SystemdPlugin, UniversalPlugin)
```

Comment:
Don't think you need `UniversalPlugin` here?

Reply:
✅